### ISSUE TITLE
Squiz.Functions.FunctionDeclarationArgumentSpacing prints wrong argument name in error message

### ISSUE THREAD

**Issue by mekras** [squizlabs/PHP_CodeSniffer#2674]
test.php:
```php
<?php

function foo($a , $b)
{
}
```
Command:

    bin/phpcs --standard=src/Standards/PSR12 test.php

Expected output:

    3 | ERROR | [x] Expected 0 spaces between argument "$a" and comma; 1 found

Actual output:

    3 | ERROR | [x] Expected 0 spaces between argument "$b" and comma; 1 found

**Comment by gsherwood** [comment]
Thanks for reporting this. The error itself was being reported and fixed correctly, but the message was wrong. This will be fixed in 3.5.3.
